src/pages/Search/SearchFiltersDatePage.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {View} from 'react-native';
+import FullPageNotFoundView from '@components/BlockingViews/FullPageNotFoundView';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Text from '@src/components/Text';
+
+function SearchFiltersDatePage() {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    return (
+        <ScreenWrapper
+            testID={SearchFiltersDatePage.displayName}
+            shouldShowOfflineIndicatorInWideScreen
+            offlineIndicatorStyle={styles.mtAuto}
+        >
+            <FullPageNotFoundView shouldShow={false}>
+                <HeaderWithBackButton title={translate('common.date')} />
+                <View style={[styles.flex1, styles.ph3]}>
+                    <Text>Advanced filters Date form</Text>
```

Comment:
Can we add a comment mentioning that this is a placeholder and a link to https://github.com/Expensify/App/issues/45026 please?